msf_core/include/msf_core/msf_sensorhandler.h

```diff
@@ -32,6 +32,10 @@ class SensorHandler {
   std::string topic_namespace_;
   std::string parameternamespace_;
   bool received_first_measurement_;
+  bool enable_mah_outlier_rejection_;
+  double mah_threshold_;
+  const double default_mah_threshold_ = 100.0;
```

Comment:
usually we define defaults outside of the class definition because clang (at least on OS X) is unhappy with this.
i.e. https://github.com/ethz-asl/rotors_simulator/blob/master/rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_imu_plugin.h#L57